lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -57,7 +57,8 @@ export async function getUpdatedPackageFiles(
     const newVersion = upgrade.newVersion!;
     const currentVersion = upgrade.currentVersion!;
     const updateLockedDependency = get(manager, 'updateLockedDependency')!;
-    packageFileManagers[packageFile] = manager;
+    packageFileManagers[packageFile] = packageFileManagers[packageFile] || [];
```

Comment:
```suggestion
    packageFileManagers[packageFile] ??= [];
```

Reply:
Applied (sorry, force-pushed because it was still on draft 😄)